DigitalExcellence/dex-frontend
# Add project images

## Description

Project images were planned for a long time in the detail page, this pull request will implement both adding and showing the project images.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project with 2 project images
2. Go to the overview page and check if the images are shown properly
3. Edit the project and remove 1 image
4. Go to the overview page and check the other image is still there.
5. Edit the project and add another image
6. Go to the overview page and check if the new image is added.

## Link to issue

Closes: #464 
Closes: #539
